### PR TITLE
Embedded video as a post

### PR DESCRIPTION
**Pull request by AdityaSher**
Embedded video as a post for visibility on mobile devices. (tested)

### CODE REVIEW

**Comment by AdityaSher** [comment]
Container size of post improper not the best solution but this is what I can do

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-livepowerfully.co
```
Comment (MandarGogate) [inline CNAME]: why did you delete this?

**Review by MandarGogate — COMMENTED** [review]
Did you verify all the changes locally?

File: CNAME (hunk 1 above)
Comment (AdityaSher) [reply CNAME]: By mistake it was deleted maybe sorry
